Formula/sdl.rb

```diff
@@ -64,22 +64,15 @@ class Sdl < Formula
     depends_on "libtool" => :build
   end
 
+  keg_only "it conflicts with the drop-in replacement `sdl12-compat`"
+
   # SDL 1.2 is deprecated, unsupported, and not recommended for new projects.
   # Commented out while this formula still has dependents.
-  # deprecate! date: "2013-08-17", because: :deprecated_upstream
-
-  conflicts_with "sdl12-compat", because: "sdl12-compat is a drop-in replacement for sdl"
+  deprecate! date: "2013-08-17", because: :deprecated_upstream
 
   def install
-    # we have to do this because most build scripts assume that all sdl modules
-    # are installed to the same prefix. Consequently SDL stuff cannot be
-    # keg-only but I doubt that will be needed.
-    inreplace %w[sdl.pc.in sdl-config.in], "@prefix@", HOMEBREW_PREFIX
```

Comment:
This may break some users' usage of `sdl`, but changing formula to keg-only means using `HOMEBREW_PREFIX` would be broken without a manual `brew link --force`. Anyway, formula can be `disable`'d right after this PR.

Reply:
We could just fully get rid of `sdl` formula and merge it into `sdl12-compat`.

That was actually my initial ambition but I was waiting for a release to come out with bug fixes, which it now has.

Reply:
So delete `sdl.rb` and add `"sdl": "sdl12-compat"` to renames?

Reply:
I think so, yeah.

Reply:
Ignoring man pages and some `libexec` files in `sdl`, the main differences are:
- we don't have static library `libSDL.a` (`sdl12-compat` also only supports building this on Linux)
- on Linux, fully versioned dynamic library is different
  - sdl uses `libSDL-1.2.so.0.11.4` vs sdl12-compat uses `libSDL-1.2.so.1.2.56`
  - latter is a bit odd since `libSDL-1.2.so.0` is symlinked to it

Reply:
Probably need a syntax-only PR due to audit.

Reply:
> * but could be seen as "upgrade" and users should rebuild with newer library for broken linkage if they didn't link to `lib/libSDL-1.2.so.0`.

If you are linking to a fully versioned libraries you're signing up for regular rebuilds (no one, for example, should be linking to `libstdc++.so.6.0.30`). This isn't anything to be concerned about.

The lack of a static library is odd, but we've shipped bigger breaking changes so should be fine.